### ISSUE TITLE
Is Layer Normalization in TransformerEncoderLayer2D placed in the correct position?

### ISSUE THREAD

**Issue by S-aiueo32** [Media-Smart/vedastr#46]
Hello,

I found Layer Normalization is placed after self-attention and the residual connection in `TransformerEncoderLayer2D`.
https://github.com/Media-Smart/vedastr/blob/d61776461d8a9c469d6875abd437de64b039a39e/vedastr/models/bodies/sequences/transformer/unit/encoder.py#L60-L63

I think it is a common way of basic transformers, however, [the official implementation of SATRN](https://github.com/clovaai/SATRN/blob/d4c257b2732137858827041f81761962a27f5228/src/networks/SATRN.py#L245-L260) places it before. Do you have any plans to align your code to the original one, or basis for reliability?

**Comment by ChaseMonsterAway** [comment]
Yes, we implement satrn before the code released. We will update the satrn in the future.

**Comment by S-aiueo32** [comment]
Good! I will keep this issue open for future fix about it!

**Comment by Johnson-yue** [comment]
this bug is not fixed for now ？？？？

**Comment by ChaseMonsterAway** [comment]
> this bug is not fixed for now ？？？？

We do some other jobs for now, i will fix it as soon as possible.

**Comment by Johnson-yue** [comment]
ok，thanks
